src/components/form/ErrorLabel.css

```diff
@@ -0,0 +1,14 @@
+.FormEdit {
```

Comment:
Shouldn't this use a ".ErrorLabel" class?
Is it that you wanted it to share some CSS with the actual form?

Reply:
ah, that's a typo. Will fix.

Reply:
Fixed in eb37e32